lib/index.js

```diff
@@ -34,11 +36,10 @@ try {
 try {
   settings = {
     ...settings,
-    ...JSON.parse(fs.readFileSync(settingsFile))
-  };
-} catch (e) {
+    ...yaml.load(fs.readFile(settingsFile, 'utf8'))
+};
+}catch (e) {
```

Comment:
No need to remove the spacing around the `catch` keyword.
```suggestion
} catch (e) {
```